src/style.css

```diff
@@ -1 +1,94 @@
-/* Styles go here */
+
+:root {
+  font-family: "Roboto", "Arial", sans-serif;
+  --color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+}
+
+.home {
+  max-width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
+  margin: 0 auto;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  align-items: center;
+  align-content: center;
+  padding: 0;
+  margin: 0;
+  text-align: center;
+}
+
+.nav__box {
+  margin-right: 20px;
+}
+
+.nav__box:last-child {
```

Comment:
On future, you can use .nav__box:not(:last-child) selector to not apply margin to the last element. It a bit shorten your code